onnx/backend/test/case/node/conv.py

```diff
@@ -0,0 +1,96 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import numpy as np
+
+import onnx
+from ..base import Base
+from . import expect
+
+
+class Conv(Base):
+
+    @staticmethod
+    def export():
+
+        x = np.array([[[  0.,   1.,   2.,   3.,   4.], # (1, 5, 5) input tensor
```

Comment:
This should be 4-D tensor, please check the [spec](https://github.com/onnx/onnx/blob/master/docs/Operators.md#inputs-2---3)

Reply:
Fixed.